packages/components/src/internal/components/labels/PrintLabelsModal.tsx

```diff
@@ -235,19 +238,28 @@ export class PrintLabelsModalImpl extends PureComponent<PrintModalProps & Inject
                     )}
                     <div className="top-spacing">
                         <b>
-                            Label template
+                            Label templates
```

Comment:
shouldn't this stay singular since the user is still only selecting a single label template

Reply:
agreed